### ISSUE TITLE
Design | CH 31 | Skip Wizard For In-Progress Form

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#68012]
## Issue Description
_Summary issue in CAIA ticket #64692._

### Details
As a Veteran who already has a form in progress, if I navigate to the apply online page, I don't want to see the wizard and need to click through again before finding the alert on the next screen letting me know I can continue an application.

### Staging or Production Link
- [Staging Link](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)

### Prototype Link
- [Mural User Flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1684161597392/5c1f8869cf5d03b792c8b82cf19b39dc26c486c2?sender=u0e8ac1d6d7681bb7e1b80558)

### Test Users
- vets.gov.user+24@gmail.com
- [staging test user repository](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/mvi-staging-users.csv)

### Proposed solution or next steps
Three options:
1. Get rid of that wizard all together
2. Have the info prompt that there is an application in progress appear on the wizard screen (when logged in)
3. Have veterans directed to the introduction page after the wizard directly if there is a form in progress

### Designs
Entry point designs:
- [How to apply](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A8585&mode=design&t=p6wBtQbVHexIKh2h-1)
- [Eligibility](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A9534&mode=design&t=p6wBtQbVHexIKh2h-1)
- [Reemployment](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A9797&mode=design&t=p6wBtQbVHexIKh2h-1)
- [Rapid access to employment](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A10066&mode=design&t=p6wBtQbVHexIKh2h-1)
- [Self-employment](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A10334&mode=design&t=p6wBtQbVHexIKh2h-1)
- [Long-term services](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A10602&mode=design&t=p6wBtQbVHexIKh2h-1)
- [Independent living](https://www.figma.com/file/gqBtkNOCslhEwes7TsBaQy/Ch31-VR%26E--Redesign?type=design&node-id=128%3A10870&mode=design&t=p6wBtQbVHexIKh2h-1)

---
## Tasks
- [ ] Decide on a solution approach.
- [ ] Draft a design solution.
- [ ] Obtain CAIA approval

## Acceptance Criteria
- [ ] Sketch file of solution.
- [ ] Assign to Laura Steele to create follow-on engineering ticket.
---

**Comment by sortizsh** [comment]
@Midge-dev  can you see if we need design work or if FE can take this and run?

**Comment by juliepedtke** [comment]
Chatted in standup today, requires UX to take a look with @Midge-dev to figure out best option. Thanks for taking a look, Sean!

**Comment by fiorella-io** [comment]
@steele-lm I don't see a screenshot of the wizard mentioned in the Details section or the steps to recreate it

**Comment by fiorella-io** [comment]
This ticket doesn't need design. 
When someone is signed in and has a Ch31 VR&E application in progress and they happen to be at any of the entry points of this form (e.g.[ How to apply](https://staging.va.gov/careers-employment/vocational-rehabilitation/how-to-apply/)) and they click "apply for VR&E benefits", redirect them to the form instead of the wizard.
<img width="1419" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNDZUQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--92a8626818e15807bd749de704faab73387a08d1/image.png" alt="image.png" />

**Comment by fiorella-io** [comment]
This might need design depending on the answer to [this thread](https://dsva.slack.com/archives/C01DBGX4P45/p1704318460923229). I remember that for all the entry points of the burial form, if the user is signed in and has an in-progress application, there is an alert informing them that they have an in-progress application as I was testing other applications, not all follow this pattern. 

<img width="1121" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM2VZQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c1767b066316caa848b83801d99be89eded398be/image.png" alt="image.png" />

**Comment by fiorella-io** [comment]
@steele-lm I've finished the designs and were added to this and the front-end stories

**Comment by steele-lm** [comment]
@fiorella-io Great! I updated the tasks and AC on the dev ticket. Do these designs still need to go through CAIA approval?

**Comment by fiorella-io** [comment]
@steele-lm I'm attending their office hours today after our sprint review. Do we need content review like Burial?

**Comment by steele-lm** [comment]
@fiorella-io I would ask CAIA. I don't think we need a full content review like burial given that we're reusing an existing pattern, but it's probably good to give them a heads up.
